docs/validators/security/backup.md

```diff
@@ -0,0 +1,37 @@
+<!--
+order: 5
+-->
+
+# Validator Backup
+
+## Synopsis
```

Comment:
sorry, I meant using this format (see example)

https://github.com/evmos/evmos/blob/26f06ee930261a3ea0cab0dd97a8a87edad7328d/docs/developers/json-rpc/endpoints.md?plain=1#L7

Reply:
sorry! thanks for the example, fixed.